Bottom: 0ba53e6d1fa7a75b0657212ae0d43f6f11778715
Top:    ddaf89b30f0811e14090dfbc141d9c760fa73891
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-06-15 09:46:58 +0800

Refresh of refresh-of-tuntap-introduce-tx

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index e10f739..d06ab7e 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1711,7 +1711,7 @@ out:
 	return ret;
 }
 
-static int tun_peek(struct socket *sock, bool exact)
+static int tun_peek(struct socket *sock)
 {
 	struct tun_file *tfile = container_of(sock, struct tun_file, socket);
 	struct sock *sk = sock->sk;
